unicodetools/data/ucd/dev/DoNotEmit.txt

```diff
@@ -429,6 +431,8 @@
 
 # Arabic, from text of Section 9.2 and the NamesList
 0649 0654; 0626; Hamza_Form # ARABIC LETTER ALEF MAKSURA, ARABIC HAMZA ABOVE; ARABIC LETTER YEH WITH HAMZA ABOVE
+064E 064E; 064B; Arabic_Taskil # ARABIC FATHA, ARABIC FATHA; ARABIC FATHATAN
+0650 0650; 063D; Arabic_Taskil # ARABIC KASRA, ARABIC KASRA; ARABIC KASRATAN
```

Comment:
L2/25-025 says that the replacement should be 064D not 063D, and U+064D is indeed the ARABIC KASRATAN as the comment says.

Reply:
Wow! Fixed.